### PR TITLE
feat: xr acls

### PR DESCRIPTION
**Pull request by stmosher**

### CODE REVIEW

File: mdd/python/translation/openconfig_xr/xr_acls.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+def xr_acls_ntp_program_service(self) -> None:
+    """
+    Apply NTP ACLs
+    """
+    device = self.root.devices.device[self.device_name].config
+    if self.service.oc_acl__acl.oc_acl_ext__ntp.server.config.server_acl_set or self.service.oc_acl__acl.oc_acl_ext__ntp.peer.config.peer_acl_set:
+        device.cisco_ios_xr__ntp.access_group.delete()
+
+    # Serve
```
Comment (m0lass3s) [inline mdd/python/translation/openconfig_xr/xr_acls.py:190]: I can't remember, but is serve a thing here or should it be server? I remember I did see "serve" as a legit value somewhere. If I'm wrong, ignore this.

Comment (stmosher-dev) [reply mdd/python/translation/openconfig_xr/xr_acls.py:190]: It is serve. I changed it in the model to server, but I was wrong.

**Review by m0lass3s — APPROVED** [review]

**Review by m0lass3s — DISMISSED** [review]

**Review by stmosher-dev — APPROVED** [review]

**Review by stmosher-dev — APPROVED** [review]
approved

**Review by stmosher-dev — APPROVED** [review]
approved

**Review by stmosher-dev — APPROVED** [review]
approved

**Pull request merged**